HLTrigger/JetMET/plugins/HLTScoutingPFProducer.cc

```diff
@@ -118,7 +118,7 @@ void HLTScoutingPFProducer::produce(edm::StreamID sid, edm::Event & iEvent, edm:
     }
 
     //get MET
-    Handle<reco::METCollection> metCollection;
+    Handle<reco::PFMETCollection> metCollection;
     std::auto_ptr<double> outMetPt(new double(-999));
     std::auto_ptr<double> outMetPhi(new double(-999));
     if(doMet && iEvent.getByToken(metCollection_, metCollection)){
```

Comment:
(this does not need to be fixed in this PR, but please consider addressing it in the future)

why do you do use
```
    outMetPt.reset(new double(metCollection->front().pt()));
```
instead of simply
```
    *outMetPt = metCollection->front().pt();
```
?

The effect should be the same, but it would save freeing and reallocating memory.